contracts/factory/ClaimIssuerFactory.sol

```diff
@@ -14,10 +16,17 @@ contract ClaimIssuerFactory is Ownable {
     /// @notice Event emitted when an address is blacklisted
     event Blacklisted(address indexed addr, bool blacklisted);
 
+    /// @notice Event emitted when the implementation is updated
+    event ImplementationUpdated(address indexed oldImplementation, address indexed newImplementation);
 
+    address public implementation;
     mapping(address => address) public deployedClaimIssuers;
```

Comment:
Lint issue here to correct about order of functions, other than that the contract looks good